lib/spider/configs/eval.yaml

```diff
@@ -0,0 +1,25 @@
+# @package _global_
```

Comment:
I have one issue with the configuration files: are they supposed to be relevant for all scripts? if that's true, isn't it a bit constraining (i.e. if I want to introduce a new script I don't necessarily want to use the same configuration params)?

Otherwise, what would you say about moving the configuration close to the script they are related to (even if it means duplicating the configs when the scripts support the same one)? (if we change this approach this can be done in another PR)

Reply:
no this config is just for in-context learning script, we should rename it

Reply:
Let's see how we organize this in another PR, let's not worry for now 🙂